.travis.yml

```diff
@@ -44,14 +44,13 @@ install:
       fi
     - pip install flake8 pytest pytest-cov
     - pip install mne psutil joblib
-    - pip install coverage coveralls
     - pip install NEURON
     - |
       cd $TRAVIS_BUILD_DIR
       python setup.py install
 
 script:
     - flake8 --count hnn_core
-    - py.test --cov=hnn_core hnn_core/tests/
+    - python -m py.test .  --cov=hnn_core hnn_core/tests/ --cov-report=xml
```

Comment:
Isn't it odd that you don't need to `source activate testenv` here first? Clearly the `python`-exec is from `testenv`, otherwise the tests would fail! But it's really strange that `flake8` works but `py.test` doesn't.